cyral/resource_cyral_integration_aws_iam_authn.go

```diff
@@ -0,0 +1,164 @@
+package cyral
```

Comment:
Lets also rename the file name to `resource_cyral_integration_aws_iam.go` instead (same for the test file).